### PR TITLE
remove ZODB3 as direct dependency

### PR DESCRIPTION
**Pull request by tomgross**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,7 +42,6 @@
         'transaction',
         'Persistence',
         'zExceptions',
-        'ZODB3',
```
Comment (tseaver) [inline setup.py:45]: Let's add `ZODB` and `BTrees` as an explicit dependency, since this product imports from them directly.

Comment (tomgross) [inline setup.py:45]: If we add ZODB here ZODB3 setups will cease to work, right?

Comment (tseaver) [inline setup.py:45]: Maybe, but nobody using Zope 2.13.x will bill in Products.ZCTextIndex 2.14.x, so the overlap should be effectively nil.

**Comment by tomgross** [comment]
What's the state of this?

**Comment by tseaver** [comment]
Thank you!

**Pull request merged**